tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -3,44 +3,39 @@
 # Licensed under the MIT License.
 
 import argparse
+import contextlib
+import enum
 import os
 import pathlib
+import tempfile
 import typing
 
 import onnxruntime as ort
-from .ort_format_model import create_config_from_models
+from .file_utils import files_from_file_or_dir, path_match_suffix_ignore_case
 from .onnx_model_utils import get_optimization_level
+from .ort_format_model import create_config_from_models
 
 
-def _path_match_suffix_ignore_case(path: typing.Union[pathlib.Path, str], suffix: str):
-    if not isinstance(path, str):
-        path = str(path)
-    return path.casefold().endswith(suffix.casefold())
-
+class OptimizationStyle(enum.Enum):
+    Fixed = 0
+    Runtime = 1
 
-def _onnx_model_path_to_ort_model_path(onnx_model_path: pathlib.Path, optimization_level_str: str):
-    assert onnx_model_path.is_file() and _path_match_suffix_ignore_case(onnx_model_path, ".onnx")
-    return onnx_model_path.with_suffix(".{}.ort".format(optimization_level_str))
 
+def _optimization_suffix(optimization_level_str: str, optimization_style: OptimizationStyle, suffix: str):
+    del optimization_level_str  # unused
+    return "{}{}".format(".with_runtime_opt" if optimization_style == OptimizationStyle.Runtime else "",
```

Comment:
What's the reason for keeping the param if it's unused now?

Reply:
thought we might want to include the optimization level again later. but we can add it back then